src/biosystems/wellness/cache.py

```
@@ -0,0 +1,616 @@
+
+    if sleep_today is not None:
+        if sleep_today < _SLEEP_RED:
+            on_red.append(f"Sleep score {sleep_today:.0f}%")
+        elif sleep_today < _SLEEP_AMBER:
+            on_amber.append(f"Sleep score {sleep_today:.0f}%")
+
+    if resp_rate is not None and rr_thresholds:
+        rr_red_t   = rr_thresholds.get("red")
+        rr_amber_t = rr_thresholds.get("amber")
+        if rr_red_t and resp_rate >= rr_red_t:
+            on_red.append(f"Resp rate {resp_rate:.1f} brpm (+{rr_sigma:.1f}σ)")
```

Ruff (0.15.6)

[warning] String contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)